corehq/pillows/domain.py

```diff
@@ -17,7 +17,7 @@
 def transform_domain_for_elasticsearch(doc_dict):
     doc_ret = copy.deepcopy(doc_dict)
     sub = Subscription.objects.filter(subscriber__domain=doc_dict['name'], is_active=True)
-    doc_ret['deployment'] = doc_dict.get('deployment', None) or {}
+    doc_ret['deployment'] = doc_ret.get('deployment', None) or {}
```

Comment:
@snopoke not sure i understand how this would change the outcome of that exception being raised?